src/plugins/explore/public/components/container/bottom_container/bottom_right_container/bottom_right_container.tsx

```diff
@@ -83,7 +82,6 @@ export const BottomRightContainer = () => {
   if (status === QueryExecutionStatus.READY) {
     return (
       <>
-        <ResultsSummaryPanel />
```

Comment:
please update `bottom_right_container.test.tsx`. therte is a mock in there for this component